src/triage/cli.py

```diff
@@ -436,5 +438,37 @@ def history(self, args):
         db_history(dburl=self.root.db_url)
 
 
+    @cmdmethod('password', action='store_true', help='hidden password prompt')
+    @local
+    def up(self, args):
+        (retcode, stdout, stderr) = self.local['docker']['container', 'inspect', '-f', "'{{.State.Status}}'", 'triage_db'].run(retcode=None)
+        if retcode != 0:
+            logger.info('Container does not exist')
+            if os.path.exists('database.yaml'):
+                logger.error("database.yaml already exists, which indicates you are "
+                              "already using Triage with a database and likely do not "
+                              "need to run this command. If you would like to provision "
+                              "a new database to use with Triage, remove database.yaml") 
+                return
+            if args.password:
+                password = getpass(prompt='Enter a password for your new database user: ')
+                logger.info(self.local['docker']['run', '-d', '-p', '5432:5432', '-e', 'POSTGRES_HOST=0.0.0.0', '-e', 'POSTGRES_USER=triage_user', '-e', 'POSTGRES_PORT=5432', '-e', f'POSTGRES_PASSWORD={password}', '-e', 'POSTGRES_DB=triage', '-v', 'db-data:/var/lib/postgresql/data', '--name', 'triage_db', 'postgres:12']())
```

Comment:
maybe for the future, but would it be useful to allow the user pass an argument to map the postgres data directory to (e.g., possibly allowing them to use a separate, larger drive) as well as a custom port (e.g., if they do have a local postgres running but don't want to use that for triage)